src/sentry/web/api.py

```diff
@@ -451,12 +486,6 @@ def _dispatch(self, request, helper, project_id=None, origin=None, *args, **kwar
     def _allowed_methods(self):
         return [m.upper() for m in self.http_method_names if hasattr(self, m)]
 
-    def options(self, request, *args, **kwargs):
```

Comment:
I think we still need this function around. This would also remove the need for the options hack in the decorator.

Reply:
We could leave options in but it will never be called ( which would be confusing in my opinion).

Adding the 'OPTIONS' verb  to the  allow string is preferable in my opinion to a method that doesn't do anything and is never called... and just used for generating the correct allow string.

Your call, let me know if you want to add an empty options method with some comment explaining why it is there or you want to manually adjust the allow header in the decorator.